src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -36,6 +37,7 @@ public static partial class Environment
             }
             return results;
         });
+        private static readonly bool IsMac = Interop.Sys.GetUnixName() == "OSX";
```

Comment:
I thought we were.  But I just went and looked again and it doesn't actually depend on this library.  @weshaggard, any problem with System.Runtime.Extensions taking a dependency on System.Runtime.InteropServices.RuntimeInformation?  That said, this is just one more function we're  importing from the native lib, and in support of Environment.OSVersion we're already importing a few other such ones.  Given that this is basically one-line, I'm not sure it's worth the extra dependency, even if it's valid.